packages/formation-react/src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const Sidebar = props => (
+  <div className="medium-screen:vads-u-padding-left--4">{props.children}</div>
```

Comment:
This should be wrapped in an `<aside>` instead of a `<div>`. Documentation should also be added to emphasize that the component should never be wrapped inside another HTML5 landmark like `<main>`

Reply:
Changed this to an aside. Folks generally know not to nest components like how you called out so I don't think it needs to be called out in the documentation.

Reply:
I will politely disagree that folks know not to nest `<aside>` in another HTML5 landmark. I've seen this a number of times, some recently, some less recently. I feel this is worth documenting because of the high likelihood this component will be used in templates and larger apps.